<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWidget</class>
 <widget class="QWidget" name="LobbyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="UsernameLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
font: 9pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="UidLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Uid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingBtn">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RuleBtn">
   <property name="geometry">
    <rect>
     <x>1780</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="BeanEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AddBeanBtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BeanLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>beanimage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DiamondEdit">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDiamondBtn">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DiamondLabel">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>diamondimage</string>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>180</y>
     <width>631</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ClassicModeBtn">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>640</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>经典模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SingleModeBtn">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>760</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="GifLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>461</width>
     <height>761</height>
    </rect>
   </property>
   <property name="text">
    <string>giflabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JoinRoomBtn">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>880</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>加入房间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RoomId">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>890</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Microsoft YaHei UI&quot;;
</string>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProfileLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
